frontend/src/scenes/session-recordings/apm/waterfall/TimeLineView.tsx

```diff
@@ -3,58 +3,88 @@ import { Tooltip } from 'lib/lemon-ui/Tooltip'
 import {
     calculatePerformanceParts,
     perfDescriptions,
+    PerformanceMeasures,
     perfSections,
     TimingBar,
 } from 'scenes/session-recordings/apm/waterfall/TimingBar'
 
 import { PerformanceEvent } from '~/types'
 
+export const validateForTimelineView = (
```

Comment:
Maybe it's just a naming thing but this feels like it does a lot more than just validation